libraries/Microsoft.Bot.Builder.GrammarChecker/ISyntaxModule.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Microsoft.Bot.Builder.GrammarChecker.SyntaxFeatures;
+
+namespace Microsoft.Bot.Builder.GrammarChecker
+{
+    public interface ISyntaxModule
```

Comment:
break into
IPosTagger
IDependencyParser

Reply:
if we breakdown into two interface, we should strong-type the result of Tagger, so that depenencyParsing can be more decoupled.